README.md

```diff
@@ -18,3 +18,91 @@ is created
 ### To run the tests locally
 
 1. Run `npm test`
+
+## Before building a docker image
+
+Before deployment of a new Docker container to the kubernetes cluster, check that the following are in place
+
+### Frontend artifacts
+
+The ODetect front end is built using Angular JS. Once built locally, the front end artifacts are stored in the `/Public/ODetect` folder.
+
+In case you are not building the front end locally, you may download a zip file of the front end artifacts from the shared ODetect vault on 1password and unzip to the above folder.
+
+### Smartthings Public Key
+
+The Samsung Smartthings Smartapp requires a public key to work, the public keys for the dev and prod environments are stored on the shared vault on 1password. Copy the contents of the public key into a file called `smartthings_rsa.pub`
+
+### Env files
+
+The env files contain important secrets such as Twilio credentials, Postgres credentials as well as the IP address of the Env files for the production and development deployments are stored on the 1password ODetect vault.
+
+### Redis cluster IP
+
+The local cluster IP of the redis deployment is a required parameter for the node application to open a connection to the database. To obtain this cluster IP, first confirm that the necessary deployment is running on the kubernetes cluster.
+
+`kubectl get pods`
+
+If there is no redis deployment, create a new deployment from a manifest by
+
+1. Checkout the desired git branch (main/dev) to `odetect-admin`
+
+1. Navigate to `\manifests` and apply the desired manifest to the kubernetes cluster using `kubectl apply -f desired_manifest.yaml`
```

Comment:
This is a Linux machine, right? I think that they use `/` instead of `\` for Linux paths.

Reply:
changed